index.html

```diff
@@ -0,0 +1,26 @@
zYpphgSKHnk2X+9CrvX7a1m8P4SUr/wBQYCggLpRVTnNZ/prKd2b2o9peraVqhMauYJCRgSjZvoaKSaUyEhkxjuO9AlvaSR5Ck+9eEE2N96Z5bLlJ2B9q4+ABnybUC8IJu2RUmOGfH6jRhLbJ6CpUdqoXzCgra1iGQKMQRDbap8FhBLEsLRgKvQpsR70Wj0iCR1Cgx8vUoeo9+9AMtIATsM/QUdsbU5XygDvtXotNmhuREkbMjHyuRsB8/Sj0FtHDFIWIbCbZ27UEBLAXYzJIET9qk9aU+ItGjg52Vy2xJPamyLUbqDTDcIbdybd3UGMqYmzhQwO++PSh19FNqtlG8DXE0zITdSsNlJH7VwRge31zQV1AGiPPExXfBNW5waxvdI5JGD+H+k9cA9vaquvrQ2kpicsGA5mHLjfG/ej/AAVxNNpUvwscCzQSOgkDHDoCceX759qCw57RcHaoEtmrAkjGDsMUxXESLgM4yeg7moc0YVDggnG2KAALLfJGFztvUpLaNdgua68uDW4oFCyjA2AAo3bDlUdh8qF2gGRRy0TmABFBJS0mm3jnWMEY3TmIH3G9FbTS+TzfEu/1A2rS1XlC7daKxDy0Fe65qfwmk3okjiiUXAht7bxFQsF2Zt+uTkZ/7RQLQuIRBHNa/GREO2I7PxHlRd+XGcY67+lP2vW1pbX0d/dLbrGWH50yAhDgjqemQf71W/EOvaZa3bnTbhZ7kEkNGAFHp0oAuozSXV3JGVBbxCMRjOSDjb1+VPXCvBF0ms/Halb+HHD0VsZkYY5enYYB9qV/w2sv6lxXatNkpb805B3JKjb/ANiDV1RTEyNk5GaDMdpEilny7keaRzuaTUvrue7EMQ50yfMRvy5601XV1PBbyyxlXCAnkkjK79hmhtpeXKosbWSGM9JUxtQY8I9q1GOhOMVLmUIhcPtjOe9QnPKxIAbP8UCzYHm9qIXN+9riK35OfGSX3A+QobpuebrUW9m57yTDYIlP2G1AQbjW70pwt/aR3MbHyPE3Iw+RG4P8UY03jT+qWck1qlvb8mxWZmY59gBVf6vKPHGMEucKP896zDcFIwiHkyMkfOgatX1W5v0WOVhOObzICFDAdRj/AA0ra3w5pbWZutMiC3PV7c/t9u3ttXjcEMDImV7srb1AvrsA8y3L4GQit1HvQFeArptD4ltjqDBUuIvBd8YCc+CM+m4H3q2zCYpmbnyCOgqg3ui3LzNzH671bX4c6x/WdANtPIGu7RvDY53KHdG+231U0BzUVMmkzmZD4SoX8jZY43Hb5UsWeoTKwiLcq9vnXXiPTtTgdpIbq5NrgFxE5IAHYg5OPmPnQRJOY9dx3FAxvcs68vNt/etRIaGWVwreV98HIbPWp6lMbMBQANNfzjegE9wXu5CrEeYnb60V0xsPnoBvSwbkCRpOo3JIoIUWoxS8U2wumPhAOi743wd/89aOX1vGkbSxSZTHUHrVa6pOwvYXDEMuWyPr/wDKdrS+8SzaAM3mGxHagj/Hx5KI+CPTNRpZPEJAcAEb5HSoN3Zm1lY+Iz5Oc+lQ/ipMnc4+lAwReCD5FB9MHodvvTT+HerxaXxIuWKwXv5Dg/tJOUOPrt/5Ui2NwhY85XONgTvXOa7cSMEPLk9u1B9PTsV5XH6sYNInGFsthdxvEipBdIXCgfpcHzAfLcH3NHdA1h9R4c02/mYNLcWqGT058Yb+Qai8awm64Tjuk/XazBsj0J5T/cfagU7W6ZMEk4NFLe6KpuwPvSjFcOp/VnYdf9KJQ3HMnb3NBrBOEtppOYeWNj9hSnLIBpzMpHpkVPvLgpp103MFCpgj1J2ApTa+KJyNnlY5NBHGj6lrGpvHpdpJcsoUNy4ABOTjJwKf9G4I4klhi57eO1bGD48oP8KSaLfhbbMNNF05ADyOVGOu+M/bFWVBKiJvJigS7D8MInCvq2oyPvlkgQKPucmmfT+CuFrFR4Wj28rD98y+Kfu2aJxyiT9px6t3rt4qAYBGaDWPSNHzhNLsx9LZf9q5XPDHD92Pz9F06Ttk2yZ++KmRyhV8xyK6fEKTtvQRLLQtOsLSO1s7cQ28ZJWNWOBkk9/mTW2o6VDeaRc6cp8KOdCuRvjPepyHmOc7Y6V4sOb+KBZh4A4fjVQ1tLIwABZp2y31wRUfVNI4Q0TwvjoEg8bPLzTPvjGe/wA6b2dQpJPSqK/GrURccVRWrNlLa2XAHqxJP9hQLPEBaKwiVB+W0uXPfODj/WgPhiQAkU3XNql/YyW7Ngtgq3oR0pSxJa3DwXC8rqcH0oLF/De/iW3azExXlVSsLdcjYsD6EAfzT7b3KyzNv5E2z6mqNmnbTlhuLcssg35x2o/pfHVzbWMJlhjky3M+DgkZoLi+LVQCCDWyTHqW2/tSvpWqx6hZQXCHHiDJXPSiEt4MhFYcv7j60Br4oyYwSIx/JqVDMAOvX1pdS7y6gH2qWbnICqdjQMtvP+Xk964SXQVudmwuc0KkvuRMA9BS1rWvhVfml8ONdvm30oGu61uKOOViylEBZjn03xVOGzi1C+vNQ1f/AIm6upOdj2QdlH0GBUXVNfnvj4KFkhRs8o9fnXKC+zku4oO1tL0Bra/0611GICcEOo8jqdx/uKHQXIznICgZLEjFSY9RhMYbxUZT0KkEGgE3DlA9vcxc8SeUlTnloRqNq9pBHLDKWtpv+WSenyo1qwkid722AZSPzFyPvQhb3Tb22lN4WR4xmFB0J70BjRuJhZ28MHisCDhiOlNcHEpbkQZOenLvmqmvp4GmVrNOROUZU77962sNXu7CcSQP33Rt1PtQXZaahPJIu2PfpRx70x26nvjb1qprfja0ithKY5fiB1jA2+9Y1H8Rbi6gMdralHK8oLHIHzoG7XuM/wCnOUVxJK+yR96UtR1ma8cXFwFz2jBwKVYrj89rq7cyTN1Jrjdag8jfl7CgNy6kMcwI+gocdW5emT60IZ2b9RNeBI6UH//Z" alt="Gandhi">
```

Comment:
It would be super helpful if you always give an explanation for your requirements. One of the many objectives code reviews have is to help the person being reviewed, and what better way to accomplish this than to give them a clear reason on why we are suggesting the changes 🤓 Kindly explain in your comments the reason for your requirements 😎